build_tools/cmake/iree_check_test.cmake

```diff
@@ -37,6 +37,23 @@ function(iree_bytecode_module_for_iree_check_test_and_friends)
     list(APPEND _RULE_FLAGS "--iree-llvm-target-triple=${_TARGET_TRIPLE}")
   endif()
 
+  if(CMAKE_SYSTEM_PROCESSOR STREQUAL "riscv" AND
+     RISCV_CPU STREQUAL "rv64" AND
+     _RULE_TARGET_BACKEND STREQUAL "llvm-cpu" AND
+     NOT _RULE_FLAGS MATCHES "iree-llvm-target-triple")
```

Comment:
Oof I wish we could do this in a cleaner way where we're not introspecting these flags. It's not any worse than what we're already doing for Android though

Reply:
Sorry.... Once we have the consolidated run_config.cmake we can trim this down. The latter two flags are used to check and support pre-specified llvm-target-triples before the following overwrite.